src/goMain.ts

```diff
@@ -132,7 +134,20 @@ export function activate(ctx: vscode.ExtensionContext): void {
 						},
 						provideCompletionItem: (document: vscode.TextDocument, position: vscode.Position, context: vscode.CompletionContext, token: vscode.CancellationToken, next: ProvideCompletionItemsSignature) => {
 							if (languageServerExperimentalFeatures['autoComplete'] === true) {
-								return next(document, position, context, token);
+								const promiseFromLanguageServer = Promise.resolve(next(document, position, context, token));
+								const promiseWithoutGoCode = getCompletionsWithoutGoCode(document, position);
+								return Promise.all([promiseFromLanguageServer, promiseWithoutGoCode]).then(([resultFromLanguageServer, resultWithoutGoCode]) => {
```

Comment:
>return [](start = 8, length = 6)

nit: this would be more readable if we would do an const [r1, r2] = await Promise.all(....) 
.....

Reply:
Done